src/Tribe/Attendee_Registration/View.php

```diff
@@ -157,19 +152,14 @@ public function display_attendee_registration_shortcode() {
 		wp_localize_script(
 			'event-tickets-registration-page-scripts',
 			'TribeCurrency',
-			[
-				'formatting' => json_encode( $currency )
-			]
+			[ 'formatting' => json_encode( $currency ) ]
```

Comment:
:warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: 13de83984ffad2bcf40aebc8a78584b2</sub>